crates/testing/tests/atomic_storage.rs

```diff
@@ -13,76 +13,6 @@ use rand::thread_rng;
 
 type AtomicStorage = hotshot::traits::implementations::AtomicStorage<DEntryState>;
 
-#[cfg_attr(
```

Comment:
Why delete this test?

Reply:
It's unrelated to the purpose of this PR, but when trying to modify the types in this test, I found it outdated and confirmed with the team that we could just delete it.